graphql/database/invitedUsers/utils.js

```diff
@@ -85,6 +86,8 @@ export const verifyAndSendInvite = async ({
       username: encodeURIComponent(invitingUser.username),
       personalMessage: inviterMessage,
       missionId: currentMissionId,
+      ...templateData,
+      title: templateData.title.replace('{{name}}', inviterName),
```

Comment:
Can you confirm inviterName is sanitized prior to use here, stripping out any HTML elements? No scripts will load in an email but still don't want an intentionally-manipulated email content.

Reply:
yup we sanitize on the handler of the request